examples/rich-text/index.js

```diff
@@ -76,6 +77,19 @@ class RichTextExample extends React.Component {
     this.editor = editor
   }
 
+  plugins = [
+    {
+      queries: {
+        isEmpty: editor => editor.value.document.text === '',
+      },
+    },
+    PlaceholderPlugin({
```

Comment:
Can you create a new example with the custom placeholder instead of using the `rich-text` example? I'd like to keep them all single-focused.

Reply:
You bet, I’ll get on this ASAP :)